phpunit.xml.dist

```diff
@@ -2,18 +2,22 @@
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          colors="true"
          bootstrap="vendor/autoload.php"
-         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.5/phpunit.xsd">
-  <coverage>
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.1/phpunit.xsd">
```

Comment:
PHPUnit 9 won't understand this configuration file. You need to stick with the (deprecated) format as long as you run tests on PHPUnit 9.

Reply:
I have reverted the config-file